deviantart/deviantart_scratcher.py

```diff
@@ -16,26 +15,32 @@
 from requests.adapters import HTTPAdapter
 from urllib3.util.retry import Retry
 
-sys.path.append(".")
 # First-party/Local
-import quantify  # noqa: E402
+import quantify
 
-PATH_REPO_ROOT, PATH_WORK_DIR, PATH_DOTENV, DATETIME_TODAY = quantify.setup(
-    __file__
+# Setup paths, Date and LOGGER using quantify.setup()
+sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), "..")))
```

Comment:
eventually, it served the purpose of `sys.path.append(".")` so that the module `quantify.py` path can be imported into the internal directories' codes

but if `sys.path.append(".")` works best for the server system, then I'll surely replace it.